src/webextension/widgets/dialog-box.js

```diff
@@ -36,8 +36,8 @@ export default class DialogBox extends React.Component {
             let primaryProps = {};
             if (i === 0) {
               primaryProps = {
-                theme: "primary",
-                ref: (element) => this._primaryButton = element,
+                theme: "critical",
```

Comment:
We shouldn't change this to `critical` (or `danger`) in all cases; some modals will want less-scary UI (in particular, I think the "cancel editing" dialog would want a blue button instead of a red one). I think we'd want to pass in `danger` as a prop to the `<DialogBox/>` so people who want a dangerous dialog can just pass `danger={true}` to it or something.

Reply:
@jimporter I've reverted to using `primary` by default. I'll need help in setting `danger` as a prop though.